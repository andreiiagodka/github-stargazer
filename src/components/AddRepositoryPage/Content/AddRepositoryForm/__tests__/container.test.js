import { shallow } from 'enzyme'
import configureStore from 'redux-mock-store'

import AddRepositoryFormContainer from '..'

describe('<AddRepositoryFormContainer />', () => {
  it('renders correctly', () => {
    const mockStore = configureStore([])
    const store = mockStore({})

    const wrapper = shallow(<AddRepositoryFormContainer.WrappedComponent store={store} />)
    const container = wrapper.dive().dive()

    expect(container).toMatchSnapshot()
  })
})
