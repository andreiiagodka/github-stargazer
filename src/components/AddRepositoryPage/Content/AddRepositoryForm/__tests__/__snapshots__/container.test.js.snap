// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AddRepositoryFormContainer /> renders correctly 1`] = `
<AddRepositoryFormComponent
  dirty={false}
  errors={Object {}}
  getFieldHelpers={[Function]}
  getFieldMeta={[Function]}
  getFieldProps={[Function]}
  handleBlur={[Function]}
  handleChange={[Function]}
  handleReset={[Function]}
  handleSubmit={[Function]}
  initialErrors={Object {}}
  initialTouched={Object {}}
  initialValues={
    Object {
      "name": "",
    }
  }
  isSubmitting={false}
  isValid={true}
  isValidating={false}
  registerField={[Function]}
  resetForm={[Function]}
  setErrors={[Function]}
  setFieldError={[Function]}
  setFieldTouched={[Function]}
  setFieldValue={[Function]}
  setFormikState={[Function]}
  setStatus={[Function]}
  setSubmitting={[Function]}
  setTouched={[Function]}
  setValues={[Function]}
  store={
    Object {
      "clearActions": [Function],
      "dispatch": [Function],
      "getActions": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
    }
  }
  submitCount={0}
  submitForm={[Function]}
  touched={Object {}}
  unregisterField={[Function]}
  validateField={[Function]}
  validateForm={[Function]}
  validateOnBlur={true}
  validateOnChange={true}
  validateOnMount={false}
  values={
    Object {
      "name": "",
    }
  }
/>
`;
